src/common/libflux/conf.c

```diff
@@ -45,27 +45,100 @@ struct flux_conf {
 static const char *conf_auxkey = "flux::conf_object";
 
 static struct builtin builtin_tab[] = {
-    { "lua_cpath_add",  INSTALLED_LUA_CPATH_ADD,    INTREE_LUA_CPATH_ADD },
-    { "lua_path_add",   INSTALLED_LUA_PATH_ADD,     INTREE_LUA_PATH_ADD },
-    { "python_path",    INSTALLED_PYTHON_PATH,      INTREE_PYTHON_PATH },
-    { "man_path",       INSTALLED_MAN_PATH,         INTREE_MAN_PATH },
-    { "exec_path",      INSTALLED_EXEC_PATH,        INTREE_EXEC_PATH },
-    { "connector_path", INSTALLED_CONNECTOR_PATH,   INTREE_CONNECTOR_PATH },
-    { "module_path",    INSTALLED_MODULE_PATH,      INTREE_MODULE_PATH },
-    { "rc1_path",       INSTALLED_RC1_PATH,         INTREE_RC1_PATH },
-    { "rc3_path",       INSTALLED_RC3_PATH,         INTREE_RC3_PATH },
-    { "cmdhelp_pattern",INSTALLED_CMDHELP_PATTERN,  INTREE_CMDHELP_PATTERN },
-    { "pmi_library_path",
-                        INSTALLED_PMI_LIBRARY_PATH, INTREE_PMI_LIBRARY_PATH },
-    { "shell_path",     INSTALLED_SHELL_PATH,       INTREE_SHELL_PATH },
-    { "shell_pluginpath",
-                        INSTALLED_SHELL_PLUGINPATH, INTREE_SHELL_PLUGINPATH },
-    { "shell_initrc",   INSTALLED_SHELL_INITRC,     INTREE_SHELL_INITRC },
-    { "jobtap_pluginpath",
-                        INSTALLED_JOBTAP_PLUGINPATH,INTREE_JOBTAP_PLUGINPATH },
-    { "upmi_pluginpath",INSTALLED_UPMI_PLUGINPATH,  INTREE_UPMI_PLUGINPATH },
-    { "no_docs_path",   INSTALLED_NO_DOCS_PATH,     INTREE_NO_DOCS_PATH },
-    { "rundir",         INSTALLED_RUNDIR,           NULL },
+    {
```

Comment:
Do you mean the reformatting and use of designated initailizers?  I did have to touch every entry in that array to accomplish the goal of the commit, so I figured if I use a different format (one more appropriate for the new content) then that was OK to do in one go.   If you feel strongly about it, let me know and I'll split it up.

Reply:
I do not feel strongly about it. Since the designated initializers ended up as most of the change I was just confused and thought this was a reformatting of the structure. (Whatever the main change was it seems to be lost in the other changes)

Reply:
Oh well that's a fair point! I was also struggling to get meaning out of the diff.  I'll fix.